src/coreComponents/constitutive/fluid/PVTDriver.cpp

```diff
@@ -108,9 +108,9 @@ void PVTDriver::postProcessInput()
 
   for( integer n=0; n<m_numSteps+1; ++n )
   {
-    m_table( n, TIME ) = minTime + n*dt;
-    m_table( n, PRES ) = pressureFunction.evaluate( &m_table( n, TIME ) );
-    m_table( n, TEMP ) = temperatureFunction.evaluate( &m_table( n, TIME ) );
+    m_table( n, static_cast< localIndex >(TIME) ) = minTime + n*dt;
```

Comment:
What was the issue here? Would it be solved if we instead specified `localIndex` as the base type of `columnKeys` enum in `PVTDriver.hpp` and `TriaxialDriver.hpp`?

Reply:
The compiler wasn't recognizing the implicit conversation of the enum to an int. The static_cast made the errors go away, I will test this branch with Ben's fix, without these "workarounds" as I think they shouldn't be necessary.